Luau/main.luau

```diff
@@ -0,0 +1,327 @@
+--!nonstrict
+
+
+type Apple = {
+  a: unknown
+}
+type Pair<T> = {first: T, second: T}
+type Tuple<T> = {first: T, second: T, third: T}
+
+--- Code:
+-- Example positive 
+-- success
+function _positive_success_f(x: unknown): unknown
```

Comment:
why do most names have underscores in front?

Reply:
to silence errors from the type checker.

Reply:
explain in the README